Lib/test/test_curses.py

```diff
@@ -1095,6 +1095,13 @@ def test_resizeterm(self):
         self.assertEqual(curses.LINES, lines)
         self.assertEqual(curses.COLS, cols)
 
+        with self.assertRaises(OverflowError):
+            curses.resizeterm(35000, 1)
+
```

Comment:
```suggestion
        with self.assertRaises(OverflowError):
            curses.resizeterm(35000, 1)
        with self.assertRaises(OverflowError):
            curses.resizeterm(1, 35000)
```

Just to be sure as well